client/me/purchases/manage-purchase/auto-renew-toggle/index.jsx

```diff
@@ -197,7 +197,7 @@ class AutoRenewToggle extends Component {
 			return translate( 'Auto-renew (…)' );
 		}
 
-		return isEnabled ? translate( 'Auto-renew (on)' ) : translate( 'Auto-renew (off)' );
+		return isEnabled ? translate( 'Auto-renew on' ) : translate( 'Auto-renew off' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Auto-renew is OFF' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/redesigned-domain-info-component)